index.js

```diff
@@ -607,9 +607,8 @@ WuiDom.prototype.delClassNames = function (classNames) {
  * @param {Boolean} [shouldAdd]
  */
 WuiDom.prototype.toggleClassNames = function (classNames, shouldAdd) {
-	classNames = joinArgumentsAsClassNames('', classNames);
-
-	if (arguments.length > 1) {
+	if (shouldAdd !== undefined) {
+		classNames = joinArgumentsAsClassNames('', classNames);
```

Comment:
By moving this statement inside of the if-block, don't we cause `classNames` being a string to break?
I notice from the JSDoc that that's not even a supported style, but I think it should be, no? (as it is for all other functions).